src/components/space/charts-layout/ChartsAlerts.js

```diff
@@ -31,50 +31,18 @@ const AlertsTooltip = withStyles({
 const ChartsAlerts = ({ metadata, view }) => {
   const { t } = useTranslation();
   const classes = useStyles();
-  const {
-    maxTreeLength,
-    maxTreeLengthExceeded,
-    requestedSampleSize,
-    numActionsRetrieved,
-  } = metadata;
+  const { requestedSampleSize, numActionsRetrieved } = metadata;
 
   const displayWarningMessage = () => {
     if (numActionsRetrieved === 0) {
       return (
         <Alert severity="warning" className={classes.alert}>
-          {t('This space and its subspaces do not have any actions yet.', {
+          {t('This item does not have any actions yet.', {
             view,
           })}
         </Alert>
       );
     }
-    if (maxTreeLengthExceeded) {
```

Comment:
This data was actually interesting, in case we want to cap the depth of the actions retrieved. Maybe let a comment there for a todo task (show message if depth is capped)?

Reply:
And maybe let's create a corresponding issue (for the next developer to work on it)